manifest.yml

```
@@ -72,6 +75,14 @@ files:
       - folder: AppTests
         files:
           - AppTests.swift
+  - folder: Resources
+    if: leaf
+    files:
+      - folder: Views
+        if: leaf
+        files:
+          - file: index.leaf
+            if: leaf
```

vapor/toolbox#351 will resolve the issue this tried to solve, so it can be reduced to this:
```suggestion
  - folder: Resources
    if: leaf
    files:
      - folder: Views
        files:
          - file: index.leaf
```